build-support/bin/native/README.md

```diff
@@ -0,0 +1,43 @@
+This directory houses shell scripts that bootstrap a rust toolchain for building and testing Pants
+native engine code.
+
+Besides the checked-in bootstrap scripts, this directory also houses symlinks generated on the fly
+to support third-party development environments (e.g. IDEA) that need access to an appropriate
+`$CARGO_HOME/bin` and rust toolchain sources (`src/`).
+
+The structure, then is:
+```
+build-support/bin/native/
+  *.sh                 - checked-in bootstrap scripts
+  cargo                - checked-in symlink to a wrapped cargo appropriate for Pants development
+  src/                 - rust stdlib sources (generated)
+  <remaining symlinks> - symlinks to the remaining $CARGO_HOME/bin tools (generated)
+```
+Some explanation of the checked-in bootstrsap scripts is in order:
+
++ `bootstrap_code.sh`
+
+  This generates a native engine binary resource with embedded provenance headers for use by pants
+  as a cffi target to access the native engine facilities via. It relies upon the `cargo` binary in
+  this directory.
+
++ `cargo.sh`
+
+  These serve as a replacement for a vanilla cargo, wrapping execution of an underlying pristine
```

Comment:
These? Plural? But just talking about one file? Should the header be "`cargo` and `cargo.sh`"?

Reply:
It was exactly as you say, but kept the header as-is and singularfied.